applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -683,6 +707,25 @@ int le_audio_send(uint8_t const *const data, size_t size)
 		return 0;
 	}
 
+	/* net_buf_alloc allocates buffers for APP->NET transfer over HCI RPMsg,
+	 * but when these buffers are released it is not guaranteed that the
+	 * data has actually been sent. The data might be queued on the NET core,
+	 * and this can cause delays in the audio.
+	 * When stream_sent_cb() is called the data has been sent.
```

Comment:
When the `stream_sent_cb()` is called, this means that the data has been successfully received by the NET core?

Reply:
Yes, ref:https://github.com/intel/zephyr/blob/main/subsys/bluetooth/host/conn.c#L371